src/_xoConnection.js

```diff
@@ -127,6 +127,16 @@ class XoConnection extends Xo {
     for (let n = disposers.length - 1; n > 0; ) {
       const params = disposers[n--];
       const method = disposers[n--];
+      if (method === "vm.delete") {
```

Comment:
`deleteTempResources` is a generic method, avoid doing specific actions on it.

Reply:
Should I create a helper to execute the job  in order to save the vm to delete at the end of the test ?

Reply:
Yes, a helper which execute the job and clean backups will be good.